Source/stores.cpp

```diff
@@ -348,6 +348,32 @@ bool StoreAutoPlace(Item &item, bool persistItem)
 	return AutoPlaceItemInInventory(player, item, persistItem);
 }
 
+void ScrollVendorStore(Item *itemData, int storeLimit, int idx, int selling = true)

```

Comment:
Um, `int selling`? Why not `bool`?

Reply:
> Um, `int selling`? Why not `bool`?

I think I was messing with other parameters and this must've been a leftover, nice catch ^^